### PR TITLE
K8SPXC-438 add possibility to use 32 symbols for cluster name

### PR DESCRIPTION
**Pull request by hors**
[K8SPXC-438](https://jira.percona.com/browse/K8SPXC-438)

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | passed |
| haproxy | passed |
| storage | passed |
| recreate | passed |
| scaling | passed |
| init-deploy | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| restore-to-encrypted-cluster | passed |
| limits | passed |
| monitoring | passed |
| upgrade-consistency | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| self-healing-advanced | passed |
| security-context | passed |
| affinity | passed |
| one-pod | passed |
| operator-self-healing | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| scheduled-backup | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | failed |
| demand-backup-encrypted-with-tls | passed |
| tls-issue-cert-manager-ref | passed |

File: build/pxc-configure-pxc.sh [hunk 1]
```diff
@@ -78,7 +78,9 @@ egrep -q "^[#]?wsrep_sst_donor" "$CFG" || sed '/^\[mysqld\]/a wsrep_sst_donor=\n
 egrep -q "^[#]?wsrep_node_incoming_address" "$CFG" || sed '/^\[mysqld\]/a wsrep_node_incoming_address=\n' ${CFG} 1<> ${CFG}
 sed -r "s|^[#]?server_id=.*$|server_id=1${SERVER_ID}|" ${CFG} 1<> ${CFG}
 sed -r "s|^[#]?wsrep_node_address=.*$|wsrep_node_address=${NODE_IP}|" ${CFG} 1<> ${CFG}
-sed -r "s|^[#]?wsrep_cluster_name=.*$|wsrep_cluster_name=${CLUSTER_NAME}|" ${CFG} 1<> ${CFG}
+
+CLUSTER_NAME=${CLUSTER_NAME%'-pxc'}
+sed -r "s|^[#]?wsrep_cluster_name=.*$|wsrep_cluster_name=${CLUSTER_NAME:0:28}-pxc|" ${CFG} 1<> ${CFG}
```
Comment (delgod) [inline build/pxc-configure-pxc.sh:83]: if we cut cluster name anyway, what the different 32 or 64 limit inside Operator

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:PR-632-17fa653f

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |
| storage | passed |
| self-healing | passed |
| haproxy | passed |
| self-healing-advanced | passed |
| init-deploy | passed |
| operator-self-healing | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| big-data | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| recreate | passed |
| tls-issue-self | passed |
| restore-to-encrypted-cluster | passed |
| tls-issue-cert-manager | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| demand-backup-encrypted-with-tls | passed |
| tls-issue-cert-manager-ref | passed |

**Pull request closed without merging**